backend/README.md

```diff
@@ -0,0 +1,104 @@
+# Licensure Compact System - Back End developer documentation
+
+## Looking for technical user documentation?
+[Find it here](./docs/README.md)
+
+## Introduction
```

Comment:
(I'm attaching this feedback to a random line in `README.md` to support a threaded discussion, since that would not be possible with PR-level feedback.)

For the second sprint, I would like to see you integrate a SaaS-hosted static code analysis tool (SonarCloud or something like it), so that each PR is automatically assessed at the time that it's opened. The idea is for all PRs from IA to CSG to have those run at the time of the PR—y'all are free to execute or not execute that analysis within your own fork of the repo as you see fit.